### PR TITLE
Content Fluentd Kinesis

### PR DESCRIPTION
**Pull request by peteclark-ft**
Adding a new provisioner which creates/deletes a kinesis stream for fluentd to write logs to

### CODE REVIEW

File: content-fluentd-kinesis/cloudformation/kinesis.yml [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  EnvironmentName:
+    Description: An environment name that will be prefixed to resource names
+    Type: String
+
+Resources:
+  ContentFluentdKinesisStream:
+    Type: "AWS::Kinesis::Stream"
+    Properties:
+      RetentionPeriodHours: 24
+      ShardCount: 1
```
Comment (sbuliarca) [inline content-fluentd-kinesis/cloudformation/kinesis.yml]: Based on the readme, should we also make this configurable?

**Comment by sbuliarca** [comment]
Other than the few comments, this looks very good to me. Very nice and clean work!!!

File: content-fluentd-kinesis/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,11 @@
+FROM alpine:3.7
+
+ENV ANSIBLE_HOSTS=/ansible/hosts
+
+RUN apk --update add python py-pip ansible bash ca-certificates py-mysqldb
+RUN pip install --upgrade pip boto boto3 dnspython awscli MySQL-python \
```
[the remark this answers is not in the document]
Comment (peteclark-ft) [reply content-fluentd-kinesis/Dockerfile]: Taken from the `pac-aurora-provisioner`, which did need them!

File: content-fluentd-kinesis/cloudformation/kinesis.yml (hunk 1 above)
Comment (peteclark-ft) [reply content-fluentd-kinesis/cloudformation/kinesis.yml]: I think I'll keep this simple for now - if we do decide to ramp up and start processing _all_ our logs via kinesis, then we will certainly need to tweak this shard count. Filtering annotations monitoring events, however, means we're highly unlikely to exceed the throughput of a single shard

Comment (sbuliarca) [reply content-fluentd-kinesis/cloudformation/kinesis.yml]: 👍

**Pull request merged**